### ISSUE TITLE
UI: Transactions using wrong network

### ISSUE THREAD

**Issue by mimetic** [Bit-Nation/BITNATION-Pangea-mobile#498]
## Feature / Issue
> User taps "Transactions" in wallet to see the browser with Etherscan, showing wallet transactions
> However, with a testing account on the Ropsten network, user should see Etherscan for that network but sees real ETH network.

## Acceptance criteria
- [ ] When user taps "Transactions" in a wallet, open the correct network for the user account, e.g. Ropsten or Main Ethereum Network

**Comment by prameetc** [comment]
@mimetic I think this is because we added conditional checks for Rinkeby and Main ETH Network. I guess we will need to another small check for Ropsten as well. @seland what do you say?

**Comment by mimetic** [comment]
@prameetc My mistake — we don't use Ropsten! However, I realized we don't tell the user which network we use (after they've set it up), so it's easy to forget and not know what to do next. So, I'm making a new issue to show the testing network somewhere. And to indicate the account is a testing account!

**Comment by seland** [comment]
@prameetc @mimetic Yeah, we're using only Rinkeby and Mainnet and there is actually a check for that, which cause different urls to be opened. @mimetic Are you sure it opens the same url for main and dev net?
